components/com_joomgallery/views/category/tmpl/default_head.php

```diff
@@ -60,7 +60,7 @@
       <?php echo JHTML::_('joomgallery.text', $this->category->description); ?>
     </div>
 <?php endif;
-      if($this->_config->get('jg_usercatorder')): ?>
+      if($this->_config->get('jg_usercatorder') && isset($this->order_by) && isset($this->order_dir)): ?>
```

Comment:
This change is now superfluous, as the template with the images is only displayed if images are also available.